src/ontology/mondo-edit.obo

```diff
@@ -94818,14 +94840,16 @@ id: MONDO:0005514
 name: nanophthalmia
 def: "Nanophthalmia is a severe form of microphthalmia characterized by a small eye with a short axial length, severe hyperopia, an elevated lens/eye ratio, and a high incidence of angle-closure glaucoma." [Orphanet:35612]
 subset: ordo_malformation_syndrome {source="Orphanet:35612"}
-xref: OMIM:600165 {source="Orphanet:35612", source="MONDO:superClassOf", source="DOID:10629", source="ORDO:35612/btnt"}
-xref: OMIM:609549 {source="Orphanet:35612", source="MONDO:superClassOf", source="DOID:10629", source="ORDO:35612/btnt"}
-xref: OMIM:611897 {source="Orphanet:35612", source="MONDO:superClassOf", source="DOID:10629", source="ORDO:35612/btnt"}
-xref: OMIM:615972 {source="Orphanet:35612", source="EFO:0005569", source="MONDO:superClassOf", source="DOID:10629", source="ORDO:35612/btnt"}
-xref: Orphanet:35612 {source="MONDO:equivalentTo", source="DOID:10629"}
+xref: DOID:0080634 {source="MONDO:equivalentTo"}
+xref: OMIM:600165 {source="Orphanet:35612", source="MONDO:superClassOf", source="DOID:10629", source="DOID:0080634", source="ORDO:35612/btnt"}
+xref: OMIM:609549 {source="Orphanet:35612", source="MONDO:superClassOf", source="DOID:10629", source="DOID:0080634", source="ORDO:35612/btnt"}
+xref: OMIM:611897 {source="Orphanet:35612", source="MONDO:superClassOf", source="DOID:10629", source="DOID:0080634", source="ORDO:35612/btnt"}
+xref: OMIM:615972 {source="Orphanet:35612", source="EFO:0005569", source="MONDO:superClassOf", source="DOID:10629", source="DOID:0080634", source="ORDO:35612/btnt"}
+xref: OMIMPS:600165 {source="DOID:0080634"}
```

Comment:
I think this should be MONDO:equivalentTo

Reply:
This OMIMPS is actually already equiv to MONDO_0021129 microphthalmia, but I think the equiv axiom should be moved to this term.
https://www.omim.org/phenotypicSeries/PS600165